sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -16,6 +22,8 @@
 
 ### Other Changes
 
+- Pipelines created with `createPipelineFromOptions` now retry up to a maximum of 3 times. To change this behavior, specify a `maxRetries` parameter in the `retryOptions` property on the optional parameters passed to `createPipelineFromOptions`.
```

Comment:
I actually updated the max retries default to be 10 in a recent PR. Is there a reason to bring it down to 3 again? /cc @xirzec

Reply:
As I remember, @xirzec requested me to set it to 3 retries as part of this PR. Perhaps I misunderstood?

Reply:
I think we should leave it at 10 since that is the existing behavior. I think it is a bit excessive, but we should wait until a major bump to tweak it since services may rely upon it. Since consumers can override this, I don't think it will hurt.

Reply:
Ok I’ll make it 10! Thank you!

Reply:
@sadasant for context, I increased the default max retries for the throttling policy to be 10 instead of 3 to be consistent with the exponential one (I did not look at the system one) and to be consistent with what Python does. This fixed an issue in Text Analytics test suite where we needed to retry more than 3 times. Granted, we could have done something custom for the tests without changing the prod behavior, but consistency with the exponential policy made me go with the default count increase. See https://github.com/Azure/azure-sdk-for-js/pull/19249

I agree with @xirzec! that we could lower the overall default count in a major version bump since it is a behavior change. I created a v3 wishlist for core-lro in https://github.com/Azure/azure-sdk-for-js/issues/17437, so perhaps you could create one for core-rest-pipeline too that includes this change?

Reply:
Not yet! I think 10 by default is fine for now. What would that wish list include? We can start thinking about it later.

Reply:
I started a wish list for core-rest-pipeline in https://github.com/Azure/azure-sdk-for-js/issues/19449